docs/modules/ROOT/pages/how-tos/testing/integration/debugging.adoc

```diff
@@ -0,0 +1,36 @@
+= Debugging Integration Tests
+
+In the event of an integration test scenario failing, there are a number of steps to take in order to debug the issue before rerunning the test in question. These steps are outlined below.
+
+
+.Prerequisites
+- You have a snapshot that has completed all built-time tests. These must be completed before attempting to debug the failed test.
```

Comment:
Looks like it should be `built-time` -> `build-time` based on [the glossary](https://konflux-ci.dev/docs/glossary/).

Reply:
Damn it apologies.